week-1/Homework/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,4 @@
-// Start by creating a variable `message`
+var name = "Burçak";
+var nameLength = name.length;
 
-console.log(message);
+console.log('My name is ' + name + ' and my name is ' + nameLength + ' characters long.');
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to build the message, it's cleaner and you don't need to remember to always add whitespaces before the + symbol:
```
console.log(`My name is ${name} and my name is ${nameLength} characters long.`);
```